EVENT.md

```diff
@@ -943,6 +943,16 @@ any one reporting origin.
 Strawman: 100 distinct destination sites per-{source site, reporting origin},
 applied to all unexpired sources regardless of type at source time.
 
+#### Limiting the number of unique destinations per source site
+
+To further reduce the possibility of a history reconstruction attack, the browser can also limit the number of `destination` eTLD+1s represented by source-sites.
```

Comment:
What does "represented" mean in this case?

Reply:
It means the destination sites that were specified by the source registrations. Changing this to "registered" to make it more clear